src/components/MediaItem/MediaItemTitle.tsx

```diff
@@ -11,14 +11,23 @@ type Props = {
   data: TautulliItemRow
   type: string
   parentRef: RefObject<HTMLDivElement>
+  loggedInUserId?: string
 }
 
 const topColors = ['text-yellow-300', 'text-gray-300', 'text-yellow-700']
 
-export default function MediaItemTitle({ i, data, type, parentRef }: Props) {
+export default function MediaItemTitle({
+  i,
+  data,
+  type,
+  parentRef,
+  loggedInUserId,
+}: Props) {
   const titleRef = useRef<HTMLSpanElement>(null)
   const numberRef = useRef<HTMLSpanElement>(null)
+  const isLoggedIn = String(data.user_id) == loggedInUserId
```

Comment:
We need a rank property in addition to the index because the logged in user may not be included in the top 6 users. Example: actual rank could be (for example 10), the 6th user gets replaced with the logged in users:
index of logged in user :5
rank of logged in user: 10